zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -236,7 +236,7 @@ trait Sql {
   // WHERE baz > buzz
   sealed case class Update[A](table: Table.Aux[A], set: List[Set[_, A]], whereExpr: Expr[_, A, Boolean]) {
 
-    def set[F: Features.IsSource, Value](lhs: Expr[F, A, Value], rhs: Expr[_, A, Value]): Update[A] =
+    def set[F: Features.IsSource, Value: TypeTag](lhs: Expr[F, A, Value], rhs: Expr[_, A, Value]): Update[A] =
```

Comment:
I'd also add a `def typeTag: TypeTag[Value]` member to `Update`, because we are requiring it here but not storing it inside the `Update` structure.

Reply:
Don't we need to capture it in `Set` since `Update` could contain zero or many sets?

Reply:
Ah right, sorry. 👍

Reply:
Ah right, sorry. 👍